server/config.ts

```diff
@@ -42,6 +42,8 @@ export default {
     port: parseInt(process.env.REDIS_PORT, 10) || 6379,
     password: process.env.REDIS_AUTH_TOKEN,
     tls_enabled: get('REDIS_TLS_ENABLED', 'false'),
+    systemTokenPrefix: `systemToken-${get('REDIS_KEY', 'clientsecretrediskey', requiredInProduction)}:`,
```

Comment:
Would it be better to default to `bapv-staff` for these two lines instead? That way you'd get the behaviour locally too. I think that would be handy so if you're switching between our set of apps locally they wouldn't be using the same default and thus reusing each other's tokens.

(I guess alternatively could just add the line `REDIS_KEY="bapv-staff"` to the example `.env` config in the `README` 🤷‍♂️)